test/python/providers/test_fake_backends.py

```diff
@@ -102,9 +101,8 @@ def setUpClass(cls):
     def test_circuit_on_fake_backend_v2(self, backend, optimization_level):
         if not optionals.HAS_AER and backend.num_qubits > 20:
             self.skipTest("Unable to run fake_backend %s without qiskit-aer" % backend.backend_name)
-        job = execute(
-            self.circuit,
-            backend,
+        job = backend.run(
+            transpile(self.circuit, backend),
             optimization_level=optimization_level,
             seed_simulator=42,
             seed_transpiler=42,
```

Comment:
This, and the other test in this file, set `seed_transpiler` which ideally needs moving into the `transpile` call.

Reply:
indeed. Done in 2fa258009c9243a6342125a7de4f459f3534b89b